src/Conversion/ONNXToKrnl/Math/Reduction.cpp

```diff
@@ -530,30 +498,27 @@ struct ONNXReduceSumOpLowering : public ConversionPattern {
       alloc =
           insertAllocAndDealloc(memRefOutType, loc, rewriter, insertDealloc);
     } else {
-      MemRefBuilder createMemRef(rewriter, loc);
       SmallVector<Value, 2> allocOperands;
       for (decltype(outRank) i = 0; i < outRank; ++i) {
         if (memRefOutShape[i] < 0) {
           if (dynamicAxes) {
             // Dim size: maskVal[i] ? 1 : inputDim[i]
-            Value inputDim = createMemRef.dim(input, i);
+            Value inputDim = create.mem.dim(input, i);
             Value indexVal = rewriter.create<arith::ConstantIndexOp>(loc, i);
```

Comment:
that one can become a `create.math.constantIndex()`

Reply:
This one will become `create.math.constantIndex()` once PR https://github.com/onnx/onnx-mlir/pull/1212 is merged in.